src/main/java/com/conveyal/taui/controllers/RegionalAnalysisController.java

```diff
@@ -266,7 +343,9 @@ public static RegionalAnalysis updateRegionalAnalysis(Request request, Response
 
     public static void register () {
         get("/api/region/:regionId/regional", RegionalAnalysisController::getRegionalAnalysis, JsonUtil.objectMapper::writeValueAsString);
-        get("/api/regional/:_id/grid/:format", RegionalAnalysisController::getPercentile, JsonUtil.objectMapper::writeValueAsString);
+        // For grids, no transformer is supplied: render raw bytes or input stream rather than transforming to JSON.
+        get("/api/regional/:_id/grid/:format", RegionalAnalysisController::getRegionalResults);
+        get("/api/regional/:_id/:comparisonId/:format", RegionalAnalysisController::getProbabilitySurface, JsonUtil.objectMapper::writeValueAsString);
```

Comment:
This line looks like a duplicate of the one below.

Reply:
Must be a copy/paste mistake, thanks. This PR was not really in a final state when I submitted it.